src/theme.rs

```diff
@@ -469,6 +470,16 @@ pub trait Theme {
         )
     }
 
+    /// Returns the progress bar template.
+    fn default_progress_template(&self) -> String {
```

Comment:
@cylewitruk It will take time to refactor, but yes. It also makes sense to re-implement a spinner = `ProgressBar` + `enable_steady_ticks` + multi-progress bar support.

Reply:
Agree :) No stress though, I'm using my fork currently so I'm not blocked, I'll switch over once there's a stable impl in main :smile: Let me know if I can help with anything?